<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBoard</class>
 <widget class="QMainWindow" name="MainBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoDesk</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionLoadWorldFile"/>
    <addaction name="actionLoadDataFile"/>
    <addaction name="separator"/>
    <addaction name="actionSaveWorldFile"/>
    <addaction name="actionSaveDataFile"/>
    <addaction name="actionSaveDataFileAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionNormalSize"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionGeoreferenceImage"/>
    <addaction name="actionSetData"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open image</string>
   </property>
   <property name="toolTip">
    <string>Open an image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit GeoDesk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom &amp;in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom &amp;out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionNormalSize">
   <property name="text">
    <string>&amp;Normal size</string>
   </property>
  </action>
  <action name="actionFitToWindow">
   <property name="text">
    <string>&amp;Fit to window</string>
   </property>
  </action>
  <action name="actionSaveWorldFile">
   <property name="text">
    <string>Save &amp;world file</string>
   </property>
  </action>
  <action name="actionSaveDataFile">
   <property name="text">
    <string>&amp;Save data file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionGeoreferenceImage">
   <property name="text">
    <string>&amp;Geo-reference image</string>
   </property>
  </action>
  <action name="actionLoadWorldFile">
   <property name="text">
    <string>&amp;Load world file</string>
   </property>
  </action>
  <action name="actionSetData">
   <property name="text">
    <string>&amp;Set data</string>
   </property>
  </action>
  <action name="actionLoadDataFile">
   <property name="text">
    <string>Load &amp;data file</string>
   </property>
  </action>
  <action name="actionSaveDataFileAs">
   <property name="text">
    <string>Save &amp;data file as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainBoard</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
